### PR TITLE
test(sbb-breadcrumb, sbb-breadcrumb-group): add visual tests

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**

### CODE REVIEW

**Comment by jeripeierSBB** [comment]
A probably non-helpful comment ;-): There is a chance that testing the breadcrumb could be unstable with our current setup. The problem is that if it's getting collapsed it stays collapsed. Depending on the timing of screen size, this could lead to problems. But maybe, or hopefully, I am wrong ;)

File: src/elements/breadcrumb/breadcrumb/breadcrumb.snapshot.spec.ts [hunk 1]
```diff
@@ -3,69 +3,69 @@ import { html } from 'lit/static-html.js';
...
+    });
+
+    it('renders - DOM', async () => {
+      await expect(element).dom.to.be.equalSnapshot();
+    });
 
-    await expect(root).shadowDom.to.equalSnapshot();
+    it('renders - Shadow DOM', async () => {
+      await expect(element).shadowDom.to.be.equalSnapshot();
+    });
```
Comment (TomMenga) [inline src/elements/breadcrumb/breadcrumb/breadcrumb.snapshot.spec.ts:27]:
```suggestion
    it('renders - Shadow DOM', async () => {
      await expect(element).shadowDom.to.be.equalSnapshot();
    });
    
    testA11yTreeSnapshot();
```

File: src/elements/breadcrumb/breadcrumb/breadcrumb.snapshot.spec.ts [hunk 2]
```diff
@@ -3,69 +3,69 @@ import { html } from 'lit/static-html.js';
...
-  testA11yTreeSnapshot(html`
-    <sbb-breadcrumb href="https://example.com/test">Breadcrumb</sbb-breadcrumb>
-  `);
+    it('renders - Shadow DOM', async () => {
+      await expect(element).shadowDom.to.be.equalSnapshot();
+    });
+
+    testA11yTreeSnapshot(html`
+      <sbb-breadcrumb href="https://example.com/test">Breadcrumb</sbb-breadcrumb>
+    `);
```
Comment (TomMenga) [inline src/elements/breadcrumb/breadcrumb/breadcrumb.snapshot.spec.ts:69]:
```suggestion
```

**Review by TomMenga — APPROVED** [review]
Just a small fix
The rest is LGTM

**Comment by DavideMininni-Fincons** [comment]
> A probably non-helpful comment ;-): There is a chance that testing the breadcrumb could be unstable with our current setup. The problem is that if it's getting collapsed it stays collapsed. Depending on the timing of screen size, this could lead to problems. But maybe, or hopefully, I am wrong ;)

For now, it looks stable, but we could monitor it build after build.

**Pull request merged**